gem2rpm/foreman_plugin.spec.erb

```diff
@@ -163,6 +169,7 @@ find %{buildroot}%{gem_instdir}/<%= spec.bindir %> -type f | xargs chmod a+x
 %foreman_precompile_plugin <%= precompile_flags.strip %>
 <% end -%>
 
+
```

Comment:
I'm slightly hestitant to changing the whitespace in all templates.

Reply:
Sorry, my bad. It is gone again.